PCIe 6.0 Packet Generator and Exerciser/PCIe 6.0 Packet Generator and Exerciser/config_space_widget.h

```diff
@@ -1,11 +1,15 @@
 #pragma once
 #include <QtWidgets/QFrame>
+#include "type1_config_space/type1_config.h"
+#include "content_widget.h"
+
+
 namespace Ui {
 	class ConfigSpaceWidget : public QFrame
 	{
 		Q_OBJECT
 		public:
-			explicit ConfigSpaceWidget(QWidget* parent = nullptr);
+			explicit ConfigSpaceWidget(ContentWidget* content, QWidget* parent = nullptr);
```

Comment:
Pass the Type1Config directly. It can be directly accessed from content widget without a getter.